edge_orchestrator/edge_orchestrator/infrastructure/metadata_storage/gcp_metadata_storage.py

```diff
@@ -9,27 +9,28 @@
 
 
 class GCPMetadataStorage(MetadataStorage):
-    def __init__(self, prefix: str):
+    def __init__(self, prefix: str, active_config_name: str):
         self.storage_client = storage.Client()
 
         self.prefix = prefix
+        self.active_config_name = active_config_name
         self.bucket = self.storage_client.get_bucket(os.getenv("GCP_BUCKET_NAME"))
 
-    def save_item_metadata(self, item: Item, active_config_name: str):
+    def save_item_metadata(self, item: Item):
         item_metadata = json.dumps(item.get_metadata())
-        blob = self.bucket.blob(os.path.join(self.prefix, active_config_name, item.id, "metadata.json"))
+        blob = self.bucket.blob(os.path.join(self.prefix, self.active_config_name, item.id, "metadata.json"))
         blob.upload_from_string(item_metadata, content_type="application/json")
 
-    def get_item_metadata(self, item_id: str, active_config_name: str) -> Dict:
-        filename = os.path.join(self.prefix, active_config_name, item_id, "metadata.json")
+    def get_item_metadata(self, item_id: str) -> Dict:
+        filename = os.path.join(self.prefix, self.active_config_name, item_id, "metadata.json")
```

Comment:
on ne peut pas suivre à peu près la même logique pour les binaries, un truc comme ça plus haut (pas pour ici):
```suggestion
        filename = os.path.join(self.prefix, self.active_config_name, item_id)
        list_blob(filename)
```